src/lib/ui/containers/SigninForm/SigninForm.svelte

```diff
@@ -53,36 +71,63 @@
         name: "password",
         type: "password",
       },
+      ...(remember
+        ? ([
+            {
+              name: "remember",
```

Comment:
🚫 **[eslint]** <[@typescript-eslint/indent](https://typescript-eslint.io/rules/indent)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected indentation of 12 spaces but found 14.
```suggestion
            name: "remember",
```